profiles/sssd/README

```diff
@@ -106,6 +106,10 @@ with-libvirt::
     Enable connecting to libvirt VMs using the hostname configured in the
     guest OS or, as a fallback, their name.
 
+with-tlog::
+    Enable support for automatic tlog session recordings in cooperation with
```

Comment:
I would remove "automatic", it's misleading, imo.

Reply:
Done.